tests/unit_tests/test_gameconnection.py

```diff
@@ -147,16 +152,35 @@ def test_abort(game_connection, game, players):
 
         assert (game.id,) == await cursor.fetchone()
 
+
 async def test_handle_action_GameResult_victory_ends_sim(game, game_connection):
     game_connection.ConnectToHost = CoroMock()
     await game_connection.handle_action('GameResult', [0, 'victory'])
 
     assert game_connection.finished_sim
     assert game.check_sim_end.called
 
+
 async def test_handle_action_GameResult_draw_ends_sim(game, game_connection):
     game_connection.ConnectToHost = CoroMock()
     await game_connection.handle_action('GameResult', [0, 'draw'])
 
     assert game_connection.finished_sim
     assert game.check_sim_end.called
+
+
+async def test_handle_action_OperationComplete(game, game_connection):
```

Comment:
IMHO we should write what do we test. Yes, we can talk about right name of the test, but we won't explain to everybody what test really does and why.